components/brave_wallet_ui/components/desktop/buy-send-swap-deposit-nav/buy-send-swap-deposit-nav.tsx

```diff
@@ -15,9 +15,14 @@ import { BuySendSwapDepositButton } from './buy-send-swap-deposit-button/buy-sen
 import { Wrapper } from './buy-send-swap-deposit-nav.style'
 
 export const BuySendSwapDepositNav = () => {
+  const buttonOptions = React.useMemo(() => {
+    // Transactions is not a option for Desktop.
+    return BuySendSwapDepositOptions.filter((option) => option.id !== 'transactions')
+  }, [])
```

Comment:
This can be computed outside of the component.

Reply:
Moved!